core/src/main/java/com/nerds/gamejam/ecs/system/BootstrapSystem.java

```diff
@@ -0,0 +1,30 @@
+package com.nerds.gamejam.ecs.system;
+
+import com.artemis.BaseSystem;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.nerds.gamejam.ecs.component.BackgroundComponent;
+import com.nerds.gamejam.ecs.component.PositionComponent;
+import com.nerds.gamejam.ecs.component.SpriteComponent;
+
+public class BootstrapSystem extends BaseSystem {
+    private static final Texture BACKGROUND_TEXTURE = new Texture("stars.png");
+
+    {
```

Comment:
What is this pair of brackety-bois doing here? :stuck_out_tongue:

Reply:
Marko told me this is how you make a static initialiser

Reply:
Ok, had to do some google :stuck_out_tongue: I forgot this even existed... what you've got right now is an instance initialiser though, not a static one though (static ones have 'static' in front of them). I think if you do this in a regular constructor and it will behave exactly the same as what you've got now. Either that, or you might want to make it static, but I'm not sure there's a reason for that, but I may be missing something...

Reply:
ahh yeah i forgot its static {} :-D been so long since ive used em ;-)

Reply:
we went for a static one cause you are working with static data so nicer fit with the semantics

Reply:
That makes sense then - I have completely forgotten these blocks were a thing... I just assumed Laura had a method at some point but deleted the signature line for some reason and this was what was left :rofl:

Reply:
nah my bad mislead her ;-D knew something was off but couldnt put finger on it, i blame old age ;-)

Reply:
Yeah, she's getting pretty old now :wink:

Reply:
Fixo'd